deps/uv/src/unix/fs.c

```diff
@@ -383,7 +392,7 @@ static ssize_t uv__fs_pathmax_size(const char* path) {
 #if defined(PATH_MAX)
     return PATH_MAX;
 #else
-    return 4096;
+#error "PATH_MAX undefined in the current platform"
```

Comment:
Indeed, I think this is a necessary change, although I know @bnoordhuis had mixed feelings. Mostly wanted to make sure it was clear from the beginning so there were no surprises
:angel: